pallets/avn-anchor/src/lib.rs

```diff
@@ -132,13 +151,28 @@ pub mod pallet {
             Ok(())
         }
 
-        #[pallet::weight( <T as pallet::Config>::WeightInfo::submit_checkpoint_with_identity())]
+        #[pallet::weight(<T as pallet::Config>::WeightInfo::submit_checkpoint_with_identity())]
```

Comment:
In a separate PR, can we create a `signed` version of this function please? We will need it to allow users to call this via the Gateway